### PR TITLE
Wrist subsystem

### LINKED ISSUE #7: Wrist Subsystem
This subsystem will need to include:

- Camera to detect when to intake object as well as validate object inside of intake.
- canCoder for position?

Possible but likely functions include:

- Moving the wrist up
- Moving the wrist down
- Aligning the wrist based on the arms position.

REV API Docs: https://codedocs.revrobotics.com/java/allclasses.html

### PR DESCRIPTION
**Pull request by gvaldez7206**
Closes #7

### CODE REVIEW

File: src/main/java/frc/robot/commands/WristAlignment.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+import frc.robot.subsystems.Wrist;
+
+/**
+ * Creates a new Command for Wrist Alignment
+ */
+public class WristAlignment extends CommandBase {
+    /** Creates a new WristAlignment. */
+    Wrist wristAlignment;
+
+    public WristAlignment(Wrist wristAlignment) {
```
Comment (legoguy1000) [inline src/main/java/frc/robot/commands/WristAlignment.java:18]: rename wristAlignment, makes no sense

File: src/main/java/frc/robot/commands/WristAlignment.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.Constants;
+import frc.robot.subsystems.Wrist;
+
+/**
+ * Creates a new Command for Wrist Alignment
+ */
+public class WristAlignment extends CommandBase {
```
Comment (legoguy1000) [inline src/main/java/frc/robot/commands/WristAlignment.java:14]: How is this command different from WristMovement??

File: src/main/java/frc/robot/modules/Vision.java [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+import edu.wpi.first.networktables.NetworkTable;
+import edu.wpi.first.networktables.NetworkTableInstance;
+import frc.robot.Constants;
+
+
+/**
+ * Vision subsystem.
+ */
+
+public class Vision {
```
Comment (legoguy1000) [inline src/main/java/frc/robot/modules/Vision.java:12]: Delete this

File: src/main/java/frc/robot/RobotContainer.java [hunk 4]
```diff
@@ -69,6 +77,10 @@ public RobotContainer() {
      * {@link edu.wpi.first.wpilibj2.command.button.JoystickButton}.
      */
     private void configureButtonBindings() {
+        operator.y().whileTrue(new WristAlignment(wrist));
+        operator.x().whileTrue(new WristMovement(wrist, 0));
+        operator.a().whileTrue(new WristMovement(wrist, -0));
```
Comment (legoguy1000) [inline src/main/java/frc/robot/RobotContainer.java:82]: whats the difference?

**Review by legoguy1000 — CHANGES_REQUESTED** [review]

**Review by agrinmanriv0537 — APPROVED** [review]

**Review by legoguy1000 — APPROVED** [review]

**Pull request merged**